### DISCUSSION TITLE
Windows::Win32::System::SystemServices::LABEL_SECURITY_INFORMATION isn't a Windows::Win32::System::Services::OBJECT_SECURITY_INFORMATION

### DISCUSSION THREAD

**Discussion by galenb-msft** [microsoft/win32metadata#595] [Q&A] [answered]
I am using GetNamedSecurityInfoA() to request security info from a file and I need the LABEL_SECURITY_INFORMATION, but since it is not a member of Windows::Win32::System::Services::OBJECT_SECURITY_INFORMATION it cannot be "or'd" into the securityinfo param of GetNamedSecurityInfoA()...

I don't know if this an issue with 0.17.2 crate, or in my understanding of how to use the LABEL_SECURITY_INFORMATION enum value in this context.

Any help resolving this issue is much appreciated!

**Comment by kennykerr** [comment]
Transferred to the win32 metadata repo where these definitions originate. I'm not familiar with this API, but hopefully they can help make sense of this.

**Comment by mikebattista** [comment, marked as answer]
This is fixed.
